packages/pods-core/src/builtin/OrbitPod.ts

```diff
@@ -0,0 +1,350 @@
+  static id: string = ID;
+  static description: string = "import orbit workspace members";
+
+  get config(): JSONSchemaType<OrbitImportPodConfig> {
+    return PodUtils.createImportConfig({
+      required: ["workspaceSlug", "token"],
+      properties: {
+        token: {
+          type: "string",
+          description: "orbit personal access token",
+        },
+        workspaceSlug: {
+          type: "string",
+          description: "slug of workspace to import from",
+        },
+      },
+    }) as JSONSchemaType<OrbitImportPodConfig>;
+  }
+
+  /**
+   * method to fetch all the members for an orbit workspace
+   * @param opts
+   * @returns members
+   */
+  getMembersFromOrbit = async (
+    opts: OrbitImportPodCustomOpts & { link: string }
+  ): Promise<any> => {
+    const { token, workspaceSlug } = opts;
+    let { link } = opts;
+    link =
+      link.length > 0
+        ? link
+        : `https://app.orbit.love/api/v1/${workspaceSlug}/members?items=100`;
+    const headers = {
+      Authorization: `Bearer ${token}`,
+    };
+    const members: MembersOpts[] = [];
+    let next = null;
+    try {
+      const response = await axios.get(link, { headers });
+      response.data.data.forEach((member: any) => {
+        const attributes = member.attributes;
+        const { id, name, github, discord, linkedin, twitter, hn, website } =
+          attributes;
+        const slugger = getSlugger();
+        members.push({
+          name: slugger.slug(name),
+          github,
+          discord,
+          linkedin,
+          twitter,
+          hn,
+          website,
+          orbitId: id,
+        });
+        next = response.data.links.next;
+      });
+    } catch (error: any) {
+      throw new DendronError({
+        message: stringifyError(error),
+        severity: ERROR_SEVERITY.MINOR,
+      });
+    }
+    return { members, next };
+  };
+
+  /**
+   * method to parse members as notes.
+   * - creates new noteprops if note is not already there in the vault
+   * - writes in a temporary hierarchy if the note is conflicted
+   * - updates previously imported notes if there are no conflicts
+   */
+  async membersToNotes(opts: {
+    members: any;
+    vault: DVault;
+    engine: DEngineClient;
+    wsRoot: string;
+    config: ImportPodConfig;
+  }) {
+    const { vault, members, engine, wsRoot, config } = opts;
+    const conflicts: Conflict[] = [];
+    const create: NoteProps[] = [];
+    members.forEach(async (member: any) => {
+      const { orbitId, name, github, discord, linkedin, twitter, hn, website } =
+        member;
+      member = {
+        custom: {
+          ...config.frontmatter,
+          orbitId,
+          social: {
+            linkedin,
+            github,
+            twitter,
+            discord,
+            hn,
+            website,
+          },
+        },
+      };
+      let noteName = name || github || discord;
+      noteName = DNodeUtils.cleanFname(noteName);
```

Comment:
to aid in debugging, its worthwile to add a `debug` log here. eg. `this.L.debug({ ctx: "membersToNotes", name, github, discord, member, noteName });`

Reply:
we can also add the twitter handle as another fallback